pages/store/[nftClassId]/index.vue

```diff
@@ -309,6 +309,39 @@
         </ul>
       </section>
 
+      <section
+        v-if="recommendedItemInfos.length"
+        class="w-full mt-8 overflow-hidden"
+      >
+        <div class="relative w-full max-w-[1200px] mx-auto">
+          <h2
+            class="text-green-500 text-lg font-bold"
+            v-text="$t('product_page_related_books_title')"
+          />
+
+          <div class="mt-6">
+            <ul
+              :class="[
+                ...gridClasses,
+                'flex-wrap', 'flex', 'gap-x-6', 'gap-y-10',
+              ]"
+            >
+              <BookstoreItem
+                v-for="(item, index) in recommendedItemInfos"
+                :id="item.classId"
+                :key="item.classId"
+                :class="getGridItemClassesByIndex(index)"
+                :nft-class-id="item.classId"
+                :book-name="item.name"
+                :book-cover-src="getResizedNormalizedImageURL(item.image || '', { size: 300 })"
+                :lazy="true"
+                @open="handleRecommendedBookCoverClick"
+              />
+            </ul>
+          </div>
+        </div>
+      </section>
```

Comment:
Simplify DOM structure and adjust margin top for mobile & desktop
```suggestion
      <section
        v-if="recommendedItemInfos.length"
        class="w-full max-w-[1200px] mx-auto mt-16 laptop:mt-20"
      >
        <h2
          class="text-green-500 text-lg font-bold"
          v-text="$t('product_page_related_books_title')"
        />

        <ul
          :class="[
            ...gridClasses,
            'flex-wrap',
            'flex',
            'gap-x-6',
            'gap-y-10',
            'mt-6',
          ]"
        >
          <BookstoreItem
            v-for="(item, index) in recommendedItemInfos"
            :id="item.classId"
            :key="item.classId"
            :class="getGridItemClassesByIndex(index)"
            :nft-class-id="item.classId"
            :lazy="true"
            @open="handleRecommendedBookCoverClick"
          />
        </ul>
      </section>
```